docs/SUMMARY.md

```diff
@@ -6,6 +6,8 @@
   * [Add a Source](quickstart/add-a-source.md)
   * [Add a Destination](quickstart/add-a-destination.md)
   * [Set up a Connection](quickstart/set-up-a-connection.md)
+* [Next Steps](next-steps/README.md)  
```

Comment:
this section doesn't feel well-defined -- why not put this page in quickstart?

Reply:
That said, I removed the `Next Steps` section for now.

Reply:
Agree with both